RELEASES.rst

```diff
@@ -1669,6 +1669,9 @@ Planned
 * Add an extra module (extras/console) providing a minimal 'console' binding
   (GH-767)
 
+* Add an extra module (extras/module-node) providing a Node.js-like module
+  loading framework supporting `require.cache`, `module.loaded`, etc.
+
```

Comment:
The formatting is incorrect for RST, should be double backticks for preformatted/code. Though I've mostly avoided formatting in the RELEASES file so that it reads better as plain text.